Classes/Task/BatchResizeAdditionalFieldProvider.php

```diff
@@ -136,12 +146,21 @@ public function validateAdditionalFields(array &$submittedData, \TYPO3\CMS\Sched
             $directoryConfig = FAL::getDirectoryConfig($directory);
             if (!@is_dir($directoryConfig['basePath'] . $directoryConfig['directory'])) {
                 $result = false;
-                (new fakeSchedulerModuleController())->addMessage(
+
+                $message = GeneralUtility::makeInstance(
+                    FlashMessage::class,
                     sprintf(
                         $GLOBALS['LANG']->sL('LLL:EXT:image_autoresize/Resources/Private/Language/locallang_mod.xlf:msg.invalidExcludeDirectories'),
                         $directory
                     ),
-                    FlashMessage::ERROR);
+                    'EXT:image_autoresize: invalidExcludeDirectories error',
+                    ContextualFeedbackSeverity::ERROR,
```

Comment:
Same here

Reply:
Yes, can be done, i leave that up to you :-)